action/template.go

```diff
@@ -15,28 +16,34 @@ import (
 	"gopkg.in/yaml.v2"
 )
 
+var err error
+
 //GenerateTemplate evaluates a template and writes it to an io.Writer
 func GenerateTemplate(out io.Writer, in io.Reader, vals interface{}) {
 	tpl, err := ioutil.ReadAll(in)
 	if err != nil {
 		log.Die("Failed to read template file: %s", err)
 	}
 
-	if err := renderTemplate(out, string(tpl), vals); err != nil {
+	if err = renderTemplate(out, string(tpl), vals); err != nil {
 		log.Die("Failed: %s", err)
```

Comment:
@smothiki can you make this say something more descriptive, like "Template rendering failed (%s)"?